src/tilesource.js

```diff
@@ -567,6 +570,20 @@ $.TileSource.prototype = {
     },
 
     /**
+     * Responsible for retrieving the headers which will be attached to the image request for the
+     * region specified by the given x, y, and level components.
+     * This option is only relevant if {@link OpenSeadragon.Options}.loadTilesWithAjax is set to true.
```

Comment:
It should be mentioned here that these headers will override headers specified at the Viewer or TiledImage level.